lib/ansible_tower_client/api.rb

```diff
@@ -96,12 +96,13 @@ def system_job_templates
     end
 
     def method_missing(method_name, *args, &block)
+      require 'faraday'
```

Comment:
Why `require 'faraday'` here?

Reply:
It's for the exception below 👇

Reply:
That won't actually work for `def/rescue/end` because those constants wouldn't exist at method definition time unless faraday is already required somewhere else.  I think it would work only for `def/require/begin/rescue/end/end`.